graphviz2drawio/graphviz2drawio.py

```
@@ -13,7 +13,7 @@ def convert(graph_to_convert: AGraph | str | IO, layout_prog: str = "dot") -> st
         graph = graph_to_convert
     else:
         graph = AGraph(graph_to_convert)
-
+    
```

Ruff (0.8.0)

Blank line contains whitespace

Remove whitespace from blank line

(W293)